### ISSUE TITLE
Loss drop little

### ISSUE THREAD

**Issue by cscyangj** [tegg89/SRCNN-Tensorflow#24]
It seems that loss drop limit during training, just 0.03% improve.

How can we make loss drop more quickly?

**Comment by PrathamSoni** [comment]
Maybe you should train more with a more aggressive learning rate.

**Comment by MashuPichu** [comment]
Use bigger learning rate for more drastic changes.
